sdk/communication/azure-communication-sms/README.md

```diff
@@ -92,8 +92,10 @@ There are two different forms of authentication to use the Azure Communication S
 ### Send a 1:1 SMS Message
 Use the `send` or `sendWithResponse` function to send a SMS message to a single phone number.
 
-<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L68-L75 -->
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L119-L128 -->
 ```java
+SmsClient smsClient = createSmsClientUsingAzureKeyCredential();
```

Comment:
Please update line number to exclude the `SmsClient smsClient = createSmsClientUsingAzureKeyCredential();`